website/src/pages/Blog/index.tsx

```diff
@@ -161,13 +161,20 @@ const Blog: React.FC = () => {
     );
   };
 
-  const getTagsMarkup = Object.keys(tagsDistribution).map((item: string) => 
-          <StyledTab
-            label={`${item}(${tagsDistribution[item as keyof typeof tagsDistribution]})`}
-            value={item}
-            key = {item}
-          />
-  );
+  const getTagsMarkup = Object.keys(tagsDistribution).map((item: string) => {
+    if (tagsDistribution[item as keyof typeof tagsDistribution] > 1) {
```

Comment:
Should this filter be applied only for mobile or for all devices @satyapriyamishra222?

Reply:
IMO, All the devices 
cc @satyapriyamishra222

Reply:
Yes, let's use it across all devices. As we see it, mostly one timer tags are cluttering the space, let's filter them out for good.